ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +651,115 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_ocs(self):
```

Comment:
please move this function to ocs_ci/ocs/utils.py so we will be able to use it also in tests